cocos/ui/UIWebView/UIWebViewImpl-android.cpp

```diff
@@ -142,15 +142,18 @@ std::string getUrlStringByFileName(const std::string &fileName) {
     // LOGD("error: %s,%d",__func__,__LINE__);
     const std::string basePath("file:///android_asset/");
     std::string fullPath = cocos2d::FileUtils::getInstance()->fullPathForFilename(fileName);
-    const std::string assetsPath("assets/");
-
     std::string urlString;
-    if (fullPath.find(assetsPath) != std::string::npos) {
-        urlString = fullPath.replace(fullPath.find_first_of(assetsPath), assetsPath.length(), basePath);
+    if (fullPath.c_str()[0] == '/') {
```

Comment:
- may crash if `fullPath` is empty
- `fullPath[0] == '/'` is better?

Reply:
yes, will better